randovania/games/dread/logic_database/header.json

```diff
@@ -1228,7 +1228,7 @@
             },
             "SlopeClimb": {
                 "long_name": "Climb Sloped Surfaces",
-                "description": "With precise movement, Flash Shift can be used to climb Sloped surfaces.",
+                "description": "It is possible to gain height on sloped surfaces with good movement. This may require usage of techniques/items such as free-aim, Flash Shift, Spin Boost, or Phantom Cloak.",
```

Comment:
It might be slightly misleading, but it stems from the two places I applied this trick, where there's limited space above the slope. The fact that one of them had a trick on it before seems to indicate that the change itself is fine, and the confusion has more to do with the trick description. I'm not sure if there's a better way to write it without elaborating a lot more.

Reply:
Yeah, that's what I meant. The trick description could be a little misleading. I'm not sure either if there's a better way to write it, but we could also just improve it later if it proves to be a problem. I think it's fine for now.